utils/config/wallets/BaseDotsamaWallet.ts

```diff
@@ -98,6 +99,7 @@ export class BaseDotsamaWallet implements Wallet {
     const unsubscribe = this._extension.accounts.subscribe(
       (accounts: InjectedAccount[]) => {
         const accountsWithWallet = accounts.map((account) => {
+          account.address = encodeAddress(decodeAddress(account.address), 2)
```

Comment:
I see some fixed ss58 prefix

Reply:
General there should be already wrapping function for this

Reply:
yeah this won't work for statemine I guess

Reply:
you find it, what should we do here? I've seen that we have many function to decode but I haven't found one that really works